app/models/concerns/plaidable.rb

```diff
@@ -5,10 +5,18 @@ module Plaidable
     def plaid_provider
       Provider::Plaid.new if Rails.application.config.plaid
     end
+
+    def plaid_eu_provider
+      Provider::Plaid.new if Rails.application.config.plaid_eu
+    end
   end
 
   private
     def plaid_provider
       self.class.plaid_provider
     end
+
+    def plaid_eu_provider
+      self.class.plaid_eu_provider
+    end
```

Comment:
I'm fine having 2 clients like this, but the main update we'll need to make is being able to grab the correct client based on the `PlaidItem` we're dealing with so that we know how to sync the data.  For example, here:

https://github.com/maybe-finance/maybe/blob/68617514b004e71b8750f9506de2dc2a9629992a/app/models/plaid_item.rb#L59

I think in `Plaidable`, we will want something like:

```ruby
def plaid_provider_for(plaid_item)
  # `region` is not yet in the schema, so we'll have to add a migration for it and populate it when we create the `PlaidItem` for the first time
  if plaid_item.region == "EU"
    plaid_eu_provider
  else
    plaid_provider
  end
end
```

Then, in `plaid_item.rb` and other places we're referencing `plaid_provider`, we'll need to switch to `plaid_provider_for(plaid_item)`